File: Sort/Programing/VB Net/Greeting.vb
Public Class Greeting

  Private _recipient As String

  Public Sub New()
    _recipient = "Stranger"
  End Sub

  Public Sub Display()
    Dim message As String = "Hello, " & _recipient
    Console.WriteLine(message)
  End Sub

  Public Property Recipient() As String
    Get
      Return _recipient
    End Get
    Set(ByVal value As String)
      _recipient = value
    End Set
  End Property
End Class

File: Sort/Programing/C or C++/Greeting.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MyFirstApplication
{
  class Greeting
  {
    private string _recipient;

    public Greeting()
    {
      _recipient = "Stranger";
    }

    public void Display()
    {
      string message = "Hello, " + _recipient;
      Console.WriteLine( message );
    }

    public string Recipient
    {
      get { return _recipient; }
      set { _recipient = value; }
    }
  }
}
